### ISSUE TITLE
logo is null

### ISSUE THREAD

**Issue by omercibcp** [kbuffington/Georgia#98]
Error: Spider Monkey Panel v1.4.1 ({B1207B07-4043-4046-AE1A-4AAB5F5AD9F1}: Georgia v2.0.2 by Mordred)
logo is null

File: georgia-main.js
Line: 1528, Column: 4
Stack trace:
  on_playback_new_track@georgia-main.js:1528:4

**Comment by kbuffington** [comment]
Thanks. This has already been fixed. Issue is Georgia is finding an invalid image. Either a file misnamed with the wrong extension or a corrupted image.

**Comment by kbuffington** [comment]
For now you can fix it by replacing 1528 and 1529 with:
```
if (logo) {
	bandLogo = art_cache.encache(logo, path);
	invertedBandLogo = art_cache.encache(logo.InvertColours(), `${path}-inv`);
}
```

**Comment by omercibcp** [comment]
Yup thanks managed to solve the issue.Had one other thing,having a hard time downloading the record label file  through git lfs. Is there not an easier way to get it for a noob like myself because when I tried to download it from the image packs on here : https://kbuffington.github.io/Georgia/docs/installation.html it's just a pointer file.Thanks in advance !

**Comment by kbuffington** [comment]
Thanks for the heads up about the recordlabel.zip file. Somehow even though I took it out of LFS it was still pointing there. All should be good now.

**Comment by kbuffington** [comment]
Fixed in 2.0.3.
